Gems/LmbrCentral/Code/Tests/ShapeThreadsafeTest.cpp

```diff
@@ -43,7 +43,7 @@ namespace UnitTest
         for (auto& queryThread : queryThreads)
         {
             queryThread = AZStd::thread(
-                [shapeEntityId, TestPoint, numIterations, &syncThreads]()
+                [shapeEntityId, TestPoint, ExpectedDistance, numIterations, &syncThreads]()
```

Comment:
If you want something that works across clang and msvc you can use a named capture  `expectedDistance = ExpectedDistance`.

Reply:
Fixed!